src/synapse/azext_synapse/_help.py

```diff
@@ -26,7 +26,7 @@
   - name: Create a Synapse workspace

```

Comment:
Let's keep `az synapse workspace` considering we will have many command groups finally. This is clearer. This is also consistent with `az sql server`. So it will be more friendly for former DW users.